packages/zoe/src/contracts/autoswap.js

```diff
@@ -40,13 +42,27 @@ export const makeContract = harden(
 
     return zcf.addNewIssuer(liquidityIssuer, 'Liquidity').then(() => {
       const keywords = harden(['TokenA', 'TokenB', 'Liquidity']);
-      const amountMaths = zcf.getAmountMaths(keywords);
+      const amountMaths = zcf.getAmountMathsByBrand();
       keywords.forEach(assertNatMathHelpers);
+
+      function getPoolKeywords(poolAllocation, inBrand) {
```

Comment:
his name implies the keywords are different per pool. The point of params is that the autoswap has a single set of 3 Keywords specified in the dapp.

Reply:
The intent was the keywords that apply to the pool (which are different from the keywords for offers). Notice the parallel with poolAmounts vs userAmounts.  The "pool" here is the single liquidity pool.